<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quick-and-Dirty Protoboard Designer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>(C) 2010 Juhapekka Piiroinen - This is a quick-and-dirty project - http://code.google.com/p/quickanddirty</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>867</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProtoboard">
    <property name="title">
     <string>Protoboard</string>
    </property>
    <addaction name="actionPrint_Board"/>
    <addaction name="separator"/>
    <addaction name="actionSave_to_File"/>
    <addaction name="actionLoad_from_File"/>
   </widget>
   <widget class="QMenu" name="menuComponents">
    <property name="title">
     <string>Insert Components</string>
    </property>
    <addaction name="actionAdd_PDIP"/>
    <addaction name="actionResistor"/>
    <addaction name="actionTO92"/>
    <addaction name="actionTO220"/>
   </widget>
   <addaction name="menuProtoboard"/>
   <addaction name="menuComponents"/>
  </widget>
  <action name="actionPrint_Board">
   <property name="text">
    <string>Send board to Printer</string>
   </property>
  </action>
  <action name="actionSave_board_as_PNG">
   <property name="text">
    <string>Save board as PNG</string>
   </property>
  </action>
  <action name="actionSave_to_File">
   <property name="text">
    <string>Save to File</string>
   </property>
  </action>
  <action name="actionLoad_from_File">
   <property name="text">
    <string>Load from File</string>
   </property>
  </action>
  <action name="actionAdd_PDIP">
   <property name="text">
    <string>PDIP</string>
   </property>
  </action>
  <action name="actionResistor">
   <property name="text">
    <string>Resistor</string>
   </property>
  </action>
  <action name="actionTO92">
   <property name="text">
    <string>TO92</string>
   </property>
  </action>
  <action name="actionTO220">
   <property name="text">
    <string>TO220</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
